php/Blockonomics.php

```diff
@@ -84,6 +95,37 @@ public function get_temp_api_key($callback_url)
         return $responseObj;
     }
 
+    //Get list of crypto currencies supported by Blockonomics
+	public function getSupportedCurrencies() {
+        return array(
+              'btc' => array(
+                    'name' => 'Bitcoin',
+                    'uri' => 'bitcoin',
+                    'enabled' => true
+              ),
+              'bch' => array(
+                    'name' => 'Bitcoin Cash',
+                    'uri' => 'bitcoincash',
+                    'enabled' => ''
+              )
+          );
+    }
+
+	//Get list of active crypto currencies
+	public function getActiveCurrencies() {
+		// $active_currencies = array();
+        $active_currencies = $this->getSupportedCurrencies();
+		foreach ($active_currencies as $code => $currency) {
```

Comment:
This loop is hard to follow. We could clarify it by doing `if code == btc` > `continue` in the beginning of the loop

`getSupportedCurrencies` array is mixing strings and boolean for enabled. By setting enabled to false for other than BTC we can also drop more from this loop

Also dont mix tabs and spaces for indentation